ui/site/src/util.js

```diff
@@ -18,10 +18,14 @@ lichess.storage = (function() {
       });
     },
     set: function(k, v) {
-      // removing first may help http://stackoverflow.com/questions/2603682/is-anyone-else-receiving-a-quota-exceeded-err-on-their-ipad-when-accessing-local
       withStorage(function(s) {
-        s.removeItem(k);
-        s.setItem(k, v);
+        try {
+          s.setItem(k, v);
+        } catch (e) {
+          // removing first may help http://stackoverflow.com/questions/2603682/is-anyone-else-receiving-a-quota-exceeded-err-on-their-ipad-when-accessing-local
```

Comment:
No. I don't understand this change. It fixes nothing, and can only introduce new obscure bugs.